### PR TITLE
IS-113 Added HSA-ID

### PR DESCRIPTION
**Pull request by martin-lindstrom**
Attribute definition of HSA-ID attribute, along with attribute set and service entity categories.

### CODE REVIEW

**Comment by maganvill** [comment]
Looks great!

File: ELN-0606 - Entity Categories for the Swedish eID Framework.md [hunk 1]
```diff
@@ -537,6 +551,8 @@ from Service Providers that have declared this entity category in their metadata
 
 - Section 2.1, "loa3-pnr", was updated with a restriction stating that the `personalIdentityNumber` only may contain a Swedish personal identity number ("personnummer") and not a coordination number ("samordningsnummer"), if attribute release is made in `loa3-pnr` scope.
 
+- The loa3-hsaid Service Entity Category was defined in section 2.6.
+
 **Changes between version 1.4 and version 1.5:**
```
Comment (Razumain) [inline ELN-0606 - Entity Categories for the Swedish eID Framework.md]: This seems to be placed in the wrong subsection. 

I assume this should be placed under: "Changes between version 1.6 and version 1.7"

Comment (martin-lindstrom) [reply ELN-0606 - Entity Categories for the Swedish eID Framework.md]: No. The version to be is 1.6, so this is correct.

File: ELN-0604 - Attribute Specification for the Swedish eID Framework.md [hunk 2]
```diff
@@ -32,11 +32,13 @@
 
     2.5. [eIDAS Natural Person Attribute Set](#eidas-natural-person-attribute-set)
 
-    2.6. [eIDAS Legal Person Attribute Set](#eidas-legal-person-attribute-set)
+    2.6. [eIDAS Legal Person Attribute Set](#eidas-legal-person-attribute-set)
+    
+    2.7. [Natural Person Identity with HSA-ID](#natural-person-identity-with-hsa-id)
```
[the remark this answers is not in the document]
Comment (martin-lindstrom) [reply ELN-0604 - Attribute Specification for the Swedish eID Framework.md]: I'll remove the 2.6 section.

**Pull request merged**